operator/attestor/attestor.go

```diff
@@ -51,6 +52,7 @@ func createEthClient(rpcUrl string, collector *rpccalls.Collector, logger sdklog
 }
 
 type Attestorer interface {
+	core.Metricable
 	Start(ctx context.Context) error
```

Comment:
```suggestion
type Attestorer interface {
	core.Metricable

	Start(ctx context.Context) error
```

Just seems a bit better to look at.

Reply:
Done